bridge/src/main/java/io/stargate/bridge/service/SchemaHandler.java

```diff
@@ -49,27 +49,39 @@
 
 class SchemaHandler {
 
+  private static final Schema.CqlKeyspaceDescribe EMPTY_KEYSPACE_DESCRIPTION =
+      Schema.CqlKeyspaceDescribe.newBuilder().build();
+
   public static void describeKeyspace(
       DescribeKeyspaceQuery query,
       Persistence persistence,
       StreamObserver<CqlKeyspaceDescribe> responseObserver) {
-    try {
-      String decoratedKeyspace =
-          persistence.decorateKeyspaceName(query.getKeyspaceName(), GrpcService.HEADERS_KEY.get());
-      Keyspace keyspace = persistence.schema().keyspace(decoratedKeyspace);
-      if (keyspace == null) {
-        throw Status.NOT_FOUND.withDescription("Keyspace not found").asException();
-      }
-      responseObserver.onNext(buildKeyspaceDescription(keyspace));
+    String decoratedName =
+        persistence.decorateKeyspaceName(query.getKeyspaceName(), GrpcService.HEADERS_KEY.get());
+
+    Keyspace keyspace = persistence.schema().keyspace(decoratedName);
```

Comment:
`CqlKeyspaceDescribe.CqlKeyspace.getName()` will be the decorated name (we don't strip the tenant prefix).
In a way this is a feature because the client will need the decorated name if it needs to maintain a global cache, like GraphQL does.

But actually it's probably better to have a separate `getDecoratedName()` field for that, I'll make the change.

Reply:
I'll add the separate field I alluded to above, hopefully it will clarify things.